uwblueprint/jump-math
# fix: miscellaneous image bugs

## Notion ticket link

[Fix upload issue for images with spaces in file names](https://www.notion.so/uwblueprintexecs/Fix-upload-issue-for-images-with-spaces-in-file-names-d6e16104c3b14057bb8fd3054240d7e0?pvs=4)
[Investigate why we can’t upload > 3 images to one assessment](https://www.notion.so/uwblueprintexecs/Investigate-why-we-can-t-upload-3-images-to-one-assessment-87a7dcbbd4474fd3a746a036e6464714?pvs=4)


## Implementation description
* Decode preview url to extract original filepath



## Steps to test
1. Create an assessment with an image that has spaces in its filename
2. Edit the assessment (anything but the image)
3. Revisit the assessment - there should 



## What should reviewers focus on?
* Might get rid of this logic entirely in follow up PR but wanted to make this fix first


## Checklist
